app/models/user.rb

```diff
@@ -56,21 +56,41 @@
 class User < ApplicationRecord
   devise :database_authenticatable, :lockable, :validatable, :timeoutable, :trackable, :invitable, :recoverable
 
-  belongs_to :vita_partner, optional: true
   has_many :assigned_tax_returns, class_name: "TaxReturn", foreign_key: :assigned_user_id
+
+  #
+  belongs_to :vita_partner, optional: true
   has_and_belongs_to_many :supported_organizations,
            join_table: "users_vita_partners",
            class_name: "VitaPartner"
+  #
```

Comment:
I'm concerned about having vita_partner, supported_organizations, _and_ memberships all at the same time. For a PR that introduces memberships, I'd like to see it completely replace the need for `vita_partner` & `supported_organizations` (and remove them from the codebase).
I would struggle with having all these implementations that solve the same issue side by side.

Reply:
We can remove supported_organizations and vita_partner as soon as we remove the shim -- these relationships are literally only here to support the ability to use honeycrisp styled fields (that require that the field responds to the method "errors" and be a method on the parent object). We should have done a better job commenting that these relationships are deprecated.